core/src/main/java/hudson/model/Queue.java

```diff
@@ -1385,9 +1385,13 @@ public void maintain() {
                             // we need to clear lost pendings as we cannot know what work unit was on this executor
                             // while it is interrupted. (All this dancing is a result of Executor extending Thread)
                             lostPendings.clear(); // we'll get them next time around when the flag is cleared.
+                            LOGGER.log(Level.FINEST,
+                                    "Interrupt thread for executor {0} is set and we do not know what work unit was on the executor.",
+                                    e.getDisplayName());
```

Comment:
:ant: Just use `getNumber`. Currently you are printing, e.g.,

    Interrupt thread for executor Executor #1 is set and we do not know what work unit was on the executor.